ql/src/java/org/apache/hadoop/hive/ql/io/orc/WriterImpl.java

```diff
@@ -582,7 +593,7 @@ public boolean hasWriterTimeZone() {
   private abstract static class TreeWriter {
```

Comment:
Do we need to bump up the writer version? To differentiate ORC files written with row-by-row vs vector writer?